lessons/beginners/list/index.md

```diff
@@ -354,25 +389,34 @@ for cislo in range(10):
 print(mocniny_dvou)
 ```
 
+Podobným způsobem získáš seznam seznam `matka`, `babička`, `prababička`,
+`praprababička`, atd.:
+
+```python
+predkove = ['matka']
+for pocet_pra in range(10):
+    predkove.append('pra' * pocet_pra + 'babička')
```

Comment:
Někde jsem zaslechl, že v případě pořadí vyhodnocování operátorů nejsou ani nadbytečné závorky na škodu. V případě násobení a sčítání je to sice notoricky známé, seč u řetězců to zas tak intuitivní být nemusí.

Reply:
To by možná bylo dobré zmínit dřív, v lekci o operátorech.

Reply:
Viz https://github.com/pyvec/naucse.python.cz/pull/649